examples/thermostat/thermostat-common/include/thermostat-delegate-impl.h

```diff
@@ -26,7 +27,7 @@ namespace Clusters {
 namespace Thermostat {
 
 /**
- * The ThermostatDelegate class serves as the instance delegate for storing Presets related information and providing it to the
+ * The ThermostatManager class serves as the instance delegate for storing Presets related information and providing it to the
```

Comment:
Schedules as well?

Reply:
schedules are not implemented yet so maybe thats fine.